apps/crn-server/src/controllers/user.controller.ts

```diff
@@ -31,6 +36,42 @@ export default class UserController {
     await this.userDataProvider.update(id, update, { suppressConflict });
     return this.fetchById(id);
   }
+  async createActiveCampaignContact(user: UserResponse): Promise<void> {
```

Comment:
extracting this and the other active campaign methods is good, but I am not sure if this should become controller responsibility. I think we intentionally removed controllers from the handlers that do not interfere with the `Response` objects (so our public API + algolia indexers use them) because controllers responsibility was supposed to be just returning data for the FE app. Maybe just have them as functions next to your handler?

Reply:
and to be fair these methods should just be part of some ActiveCampaign client or something like that, interaction with that third party API or generating API specific payload mixed with all the remaining functionality here is imho not a great practice